client/scripts/index.js

```diff
@@ -69,18 +86,19 @@ function game_init(pad) {
 // this function exports the whiteboard as JSON and sends it to the server
 function submit_stroke_to_server(player_number, pad) {
     const newBoardState = JSON.stringify(pad.toJSON())
-    console.log(newBoardState)
     socket.emit('gameplay_stroke', {
         player_number: player_number,
         new_board_state: encodeURIComponent(newBoardState)
     })
+    disableButtons()
```

Comment:
can add a comment here, to explain in plain english
that after you hit submit, we disable all buttons for the user  until its their turn again